etc/derecho/build_tuvx_derecho_intel.sh

```diff
@@ -21,24 +21,7 @@ if [[ ! -d "${TUVX_HOME}" ]]; then
   return
 fi
 
-echo "Building JSON Fortran"
-
-# get & build the source code of JSON Fortran
-
-cd ${TUVX_HOME}
-curl -LO https://github.com/jacobwilliams/json-fortran/archive/8.3.0.tar.gz
-tar -zxvf 8.3.0.tar.gz
-cd json-fortran-8.3.0
-mkdir build
-cd build
-INSTALL_DIR=$TUVX_HOME/json-fortran-8.3.0
-cmake -D SKIP_DOC_GEN:BOOL=TRUE -D CMAKE_INSTALL_PREFIX=$INSTALL_DIR ..
-make install
-
-echo "Building TUV-x"
-
 # get & build the source code of TUV-x
-
 cd ${TUVX_HOME}
```

Comment:
@K20shores @mattldawson I think the Intel compilers on Derecho does not build all the tests successfully. Shall we remove this file to avoid confusion? Or we try to figure out what could be wrong in the code?

Reply:
My suggestion would be to make a note in `etc/derecho/README.md` that not all the tests build with the Intel compiler, and then create an issue to look into it more, just so we don't forget about it.

Reply:
That sounds good!

Reply:
I have added the note under the build section in `etc/derecho/README.md`